BusinessCentral.LinterCop/Design/Rule0054FollowInterfaceObjectNameGuide.cs

```diff
@@ -103,6 +103,9 @@ private static List<string> GetAffixes(Compilation compilation)
             if (copConfiguration is null)
                 return null;
 
+            if (copConfiguration.MandatoryAffixes == null)
```

Comment:
Could it be that the exit is to early on here? There's a Prefix (instead of Affix) in the AppSourceCop configuration these don't will be added to the list.

Reply:
As this exception was popping up at testing my changes regarding the new rule, I probably didn't look close enough. Will have a look at it again.